app.py

```
@@ -49,6 +49,7 @@ def inject_user():
 
 @app.route('/')
 def index():
+    theme = request.cookies.get("theme", "light")
```

Ruff (0.8.2)

Local variable `theme` is assigned to but never used

Remove assignment to unused variable `theme`

(F841)